src/App.js

```diff
@@ -35,21 +40,24 @@ function App() {
     </Router>
   )
 
-  function createTrip() {
-    return trip => setTrips([trip, ...trips])
+  function addTrip(trip) {
+    const newTrips = [trip, ...trips]
+    setTrips(newTrips)
+    saveToLocal('trips', newTrips)
```

Comment:
Suggestion: You can use an useEffect hook on trips and save to the localStorage as soon as anything changes. Makes it easier later on as every change will be saved.
`
useEffect(()=>{
  saveToLocal('trips', trips);
},[trips])
`